### PR TITLE
Mg 5 implement game manager

### PR DESCRIPTION
**Pull request by AnejBezjak**

### CODE REVIEW

File: Assets/Scripts/GameManager.cs [hunk 1]
```diff
@@ -19,12 +20,20 @@ void Start()
...
     }
 
     // Update is called once per frame
     void Update()
     {
-        
+        if (isGameOver)
+        {
+            // Restart game if player presses R
+            if (Input.GetKeyDown(KeyCode.R))
```
Comment (misel-cucek) [inline Assets/Scripts/GameManager.cs:36]: Does the player know if they have to press letter R for restart?

**Review by misel-cucek — CHANGES_REQUESTED** [review]
Does player know if they have to press letter R for restart

File: Assets/Scripts/GameManager.cs (hunk 1 above)
Comment (AnejBezjak) [reply Assets/Scripts/GameManager.cs:36]: No

**Review by misel-cucek — APPROVED** [review]
LGTM

**Pull request merged**